dydxprotocol/v4-web
# chore: learn more link in equity tier broadcast error

- removed FE validation for equity tier limit
- add learn more link in broadcast error for equity tier limit

![Screenshot 2024-05-29 at 2 01 19 PM](https://github.com/dydxprotocol/v4-web/assets/9400120/918fa01a-d979-4278-8237-9fd414d56251)

to test,
use an account that has low net collateral, create a bunch of orders until you see the error
